lib/seccomp/CSystemCallFilter_MacOSX.cc

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+
+#include <paths.h>
+#include <cstdio>
+#include <fstream>
+#include <sandbox.h>
```

Comment:
We generally separate the C and C++ headers with a blank line, C++ headers first.